CHANGELOG.md

```diff
@@ -6,28 +6,32 @@ The format is based on [Keep a
 Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Develop Branch]
+## [v2022.03.0 - 2022-03-14]
 
-### Added
+### Build Changes and Updates
 
-- Added a getDeviceAllocator function that allows users to get a DeviceAllocator object
-  from the kernel without explicitly passing the allocator to the kernel first.
-
-- Added a reset function to the DeviceAllocator so that old data can be rewritten.
+- Umpire now requires minimum cmake version to be 3.14
 
-- Added documentation of the DeviceAllocator.
+- Umpire now requires C++14.
 
-### Changed
+- Use C++17 for SYCL backend.
 
 - Reorganized cmake object library for c/fortran interface. NOTE: This is a breaking
-  change since the include paths are different. 
 
-- Build Doxygen documentation on ReadTheDocs.
+  change since the include paths are different.
 
-- Changed more CMakeList options to have 'UMPIRE' prefixes and made them dependent
-  on the corresponding BLT options. 
+- Changed CMakeList options to have `UMPIRE` prefixes and made them dependent
+  on the corresponding BLT options.
 
-- Umpire now requires C++14.
+- C/FORTRAN API is now auto generated
+
+- Make umpire-config.cmake package relocatable
+
+- Add `UMPIRE_ENABLE_DEVICE_ALLOCATOR` option to disable Device Allocator which defaults
+  to "Off", to control whether or not the DeviceAllocator class is included in the
+  library.
+
+- Use blt namespace for hip targets (#692)
```

Comment:
I am not a markdown expert by any means, but does a `#` symbol here cause markdown to get confused, or does it look ok?

Reply:
Good question.  It seems to be okay though (see: https://github.com/LLNL/Umpire/blob/release/2022.03.0/CHANGELOG.md)